adhocteam/Head-Start-TTADP
# [TTAHUB 280/281/282/283] Fixes for the accessible data view & line graph

## Description of change

The table now has a horizontal scroll bar to handle large data sets.
Hours are rounded to one decimal place.
The language "hours" was removed from the individual cells and placed in the row heading.
Text was added to the dummy column
Tooltips now handle numbers with decimal a little bit better

## How to test
View a large dataset with some oddball floats. Observe that the table will scroll and that any floats are rounded to one decimal place

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-280
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-281
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-282
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-283


## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
